libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -643,8 +643,9 @@ void RangeFinder::detect_instance(uint8_t instance, uint8_t& serial_instance)
     switch (_type) {
     case RangeFinder_TYPE_PLI2C:
     case RangeFinder_TYPE_PLI2CV3:
-        if (!_add_backend(AP_RangeFinder_PulsedLightLRF::detect(1, state[instance], _type))) {
-            _add_backend(AP_RangeFinder_PulsedLightLRF::detect(0, state[instance], _type));
+    case RangeFinder_TYPE_PLI2CV3HP:
+        if (!_add_backend(AP_RangeFinder_PulsedLightLRF::detect(2, state[instance], _type))) {
```

Comment:
this doesn't looks good ? why change the i2c bus?

Reply:
Hi, I changed the bus to use Beaglebone i2c2 port, I have reverted this change so it won't cause trouble.